website/blog/2025/07/07-16-enhancing-data-protection-with-immutable-backup-buckets.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Enhancing Data Protection with Immutable Backup Buckets"
+linkTitle: "Enhancing Data Protection with Immutable Backup Buckets"
+newsSubtitle: July 16, 2025
+publishdate: 2025-07-16
+authors:
+- avatar: https://avatars.githubusercontent.com/ishan16696
+  login: ishan16696
+  name: Ishan Tyagi
+aliases: ["/blog/2025/07/16/enhancing-data-protection-with-immutable-backup-buckets"]
+---
+
+Gardener has introduced support for immutable backup buckets, a critical feature for enhancing the security and resilience of your Kubernetes clusters. This new capability leverages native cloud provider features to protect your etcd backups from accidental or malicious deletion and modification, helping you meet stringent security and compliance requirements.
+
+### What Are Immutable Backup Buckets?
+
+Immutable backup buckets ensure that once a backup snapshot is written, it cannot be altered or deleted for a predefined retention period. Gardener configures this by applying immutability policies (often called "Object Lock" or "WORM" - Write Once, Read Many) to the backup buckets managed by provider extensions.
+
+When enabled, the provider extension will:
+
+- **Create** new backup buckets with the specified immutability policy.
+- **Reconcile** existing buckets to enforce the desired policy.
+- **Prevent** any changes that would weaken the policy, such as shortening the retention period or disabling the lock.
+
+It is crucial to understand that once an immutability policy is locked on a bucket, it is irreversible and cannot be shortened, even by an administrator.
+
+### How It Works
+
+The feature is configured on the `Seed` resource. When you enable immutability, only newly created backup objects will inherit the lock settings. Over time, as old, unlocked backups are rotated out, all snapshots in the bucket will be protected by the immutability policy.
```

Comment:
According to the [GCS Bucket Lock documentation](https://cloud.google.com/storage/docs/bucket-lock), the retention policy can apply **retroactively** to existing objects in the bucket, as well as to any new objects added after the policy is set.
>

Since this behavior may vary depending on the cloud provider, could you please update the documentation to reflect that the immutability policy is **provider-specific** and may or may not apply to existing objects?